qemu/tests/timedrift_check_after_load_vm.py

```diff
@@ -0,0 +1,158 @@
+import re
+import time
+import logging
+
+from avocado.utils import process
+from virttest import env_process
+from virttest import error_context
+from virttest import guest_agent
+from virttest import arch
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Clock check after savevm and loadvm
+    Note: Run this test case, make sure that guest agent is installed
+    and start inside the guest.
+
+    1. On host, load kvm module with "kvmclock_periodic_sync=N"
+    2. On host, sync time with ntp server
+    2. Boot guest with qemu guest agent
+    3. On guest, sync time with ntp server
+    4. Run samevm and loadvm
```

Comment:
Sorry @dinghuiqing , there is an update on polarion case, could you help stop the `chronyd` service before doing `savevm`? The `chronyd` may mislead this case's result.